### PR TITLE
Edit locations

### PR DESCRIPTION
**Pull request by dwhenry**

### CODE REVIEW

File: app/assets/stylesheets/admin/components/_inline-label.scss [hunk 1]
```diff
@@ -0,0 +1,5 @@
+.label--inline {
```
Comment (benbarnett) [inline app/assets/stylesheets/admin/components/_inline-label.scss]: This file should just be called `label` now as it should be named after the block

File: app/assets/stylesheets/admin/application.scss [hunk 2]
```diff
@@ -3,6 +3,7 @@
 @import "./lib/variables";
 @import "./helpers/contain-floats";
 
+@import "./components/inline-label";
```
Comment (benbarnett) [inline app/assets/stylesheets/admin/application.scss]: Same here re filename

File: features/pages/admin_edited_locations_page.rb [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+
+  element :previous_day, '.t-previous-day'
+
+  sections :locations, '.t-location' do
+    element :location_title, '.t-location-title'
+
+    sections :edits, '.t-edit' do
+      element :field, '.t-edited-field'
+      element :from, '.t-edited-from'
+      element :to, '.t-edited-to'
```
Comment (benbarnett) [inline features/pages/admin_edited_locations_page.rb]: Should we have coverage here for who it was edited by?

File: spec/models/edited_location_field_spec.rb [hunk 4]
```diff
@@ -0,0 +1,85 @@
...
+      let(:previous) { location(title: 'Alpha') }
+
+      it 'returns an edit record for each field that has changed' do
+        results = subject.all(current, previous)
+        expect(results.count).to eq(1)
+      end
+
+      context 'returned object is correctly populated can on edit details' do
+        it 'for the field that is changed' do
+          results = subject.all(current, previous)
```
Comment (benbarnett) [inline spec/models/edited_location_field_spec.rb]: Is there a `subject.first`?

Comment (dwhenry) [inline spec/models/edited_location_field_spec.rb]: not at this stage as you are getting all the differences (edits) between the two locations that are passed in.

File: features/pages/admin_edited_locations_page.rb (hunk 3 above)
Comment (dwhenry) [inline features/pages/admin_edited_locations_page.rb]: added

Comment (dwhenry) [inline features/pages/admin_edited_locations_page.rb]: changed both :)

**Comment by benbarnett** [comment]
:+1:

**Pull request merged**